### ISSUE TITLE
minor visual glitch when resizing the shell window via keyboard navigation

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#13552]
**Setup**
- Rancher version: master
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

There's a minor visual glitch when resizing the shell window via keyboard navigation that should be fixed. First reported in https://github.com/rancher/dashboard/pull/13519#pullrequestreview-2651135209

**To Reproduce**

**Result**
interaction results in minor visual glitch

**Expected Result**

no visual glitches should happen

**Screenshots**

**Additional context**

**Comment by rak-phillip** [comment]
The issue described above is minor enough that we don't need to fix this. If somebody volunteers to address, we can accept contributions.
